### ISSUE TITLE
allow voters to see how the delegate votes before votes are tallied

### ISSUE THREAD

**Issue by GriffGreen** [win2win/LiquidDemocracy#1]
One feature that would be a really nice would be if the delegate had to lock in their bid for a certain amount of time before being able to vote.

That way if a voter didn't like how their delegate was voting, they could actually change their delegate and either vote for themselves or find a delegate that will vote the way they want 

This is really cool win2win, nice work :-D

**Comment by win2win** [comment]
Thanks for commenting.

If I understand correctly you are suggesting option which would allow a member to cancel the delegation during the voting period and vote themselves if they didn't like the direction their delegate was taking?

There are functions available in the product already which allow to determine how member or deleage has voted and also to see the current totals of voting before the tallying. 

function hasVoted
function numOfVotes
function howVoted
function hasDelegateVoted
function howDelegateVoted
function calculateVotes

**Comment by GriffGreen** [comment]
Yep! Exactly, ideally it would require a delegate to commit to a vote ~2 days before hand and then the voter could check and make sure that they agree with that vote, if not they can switch their delegate and their vote
